ibis/backends/bigquery/client.py

```diff
@@ -72,6 +72,23 @@ def bigquery_schema(table):
     return sch.schema(fields)
 
 
+def ibis_schema_to_bigquery_schema(schema: sch.Schema):
```

Comment:
Hm, I thought we had this function somewhere already.

Reply:
I guess not!

Reply:
We have a function that allows us to convert in the opposite direction only.

Reply:
Yep. I'll be putting up a PR in the next few days to clean up all the to_x/from_x ibis types/schemas.